easybuild/easyconfigs/p/PyGWAS/PyGWAS-1.3.1-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = "PythonPackage"
+
+name = 'PyGWAS'
+version = '1.3.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/timeu/pygwas'
+description = """PyGWAS is a library for running Genome Wide Association studies."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2fd4264b92697efab7a09f815f004015554b3e46c74c81d8ed1bec8d2f20f9c9']
```

Comment:
@timeu This is the same checksum as for version `1.2.0`, that can't be right...

Copy-paste fail?

Easiest way to fix is to use `eb PyGWAS*eb --inject-checksums --force`.

Reply:
That should be fixed by the last commit. 
Are there any changes that I missed ?